font-src/glyphs/number/0.ptl

```diff
@@ -31,16 +34,65 @@ glyph-block Digits-Zero : begin
 	define [ZeroOvalMask top] : ZeroOvalShapeT spiro-outline 1 1 top
 
 
+	define [InsetShape zero overlay fnMask height] : glyph-proc
+		include zero
+		if fnMask
+			include : intersection overlay
+				WithTransform [ApparentTranslate (-SplitSlashGap) 0] [fnMask height]
+				WithTransform [ApparentTranslate   SplitSlashGap  0] [fnMask height]
+			include overlay
+
+	define [Cutout zero overlay fnMask height] : glyph-proc
+		if overlay
+			include : difference zero overlay
+			include zero
+
+
+	define [CutoutSlash top] : dispiro
+		flat 0 (top * (1 - 0.77)) [widths.center (top * 0.125)]
```

Comment:
Change the stroke width after (`widths.center`) to things like
`[Math.min (0.4 * CircleInnerWidth) (top * 0.15) [AdviceStroke 3]]`

Adjust the parameters to make the SW being proper under all the weights.